typescript/common-resolvers/src/import/format-coco/converters.ts

```diff
@@ -52,18 +52,30 @@ export const isCocoSegmentationBox = (
 
 export const convertCocoSegmentationToLabel = (
   cocoSegmentation: number[][],
+  bbox: number[], // [x, y, width, height]
   imageHeight: number
 ): { geometry: GeometryInput; type: LabelType } => ({
   type: isCocoSegmentationBox(cocoSegmentation)
     ? LabelTypeOptions.Box
     : LabelTypeOptions.Polygon,
   geometry: {
     type: "Polygon",
-    coordinates: cocoSegmentation.map((cocoPolygon) =>
-      range(0, cocoPolygon.length / 2).map((index) => [
-        cocoPolygon[2 * index],
-        imageHeight - cocoPolygon[2 * index + 1],
-      ])
-    ),
+    coordinates:
+      !cocoSegmentation || cocoSegmentation.length === 0
+        ? [
+            [
+              [bbox[0], bbox[1] + bbox[3]],
+              [bbox[0] + bbox[2], bbox[1] + bbox[3]],
+              [bbox[0] + bbox[2], bbox[1]],
+              [bbox[0], bbox[1]],
+              [bbox[0], bbox[1] + bbox[3]],
+            ],
+          ]
```

Comment:
Ok, then I'm not sure from where X and Y originate here.
If it's from the top-left, I am going *counter*-clockwise, but if XY originate from bottom left, it's indeed clockwise.

If my geometry is wrong, we might have another issue, because I tried to keep the same logic/order as the data I got from exporting the tutorial dataset to COCO.
```json
    {
      "id": 2,
      "image_id": 3,
      "category_id": 1,
      "segmentation": [
        [
          0, 1554.7676719276355,
          174.8702716326667, 1554.7676719276355,
          174.8702716326667, 1401.2061213908169,
          0, 1401.2061213908169,
          0, 1554.7676719276355
        ]
      ],
      "area": 26853.350054706956,
      "bbox": [0, 1401.2061213908169, 174.8702716326667, 153.5615505368187],
      "iscrowd": 0
    }
```

Reply:
Yes X and Y originate from bottom-left (and both have only positive values), we should write it somewhere once for all not to get confused. 
And yes indeed you are right, I just checked and it seems like when we create bounding boxes, we do them the wrong way, while polygons are always created counter clock-wise. So it must be bounding box specific code that is wrong.